<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCardDlg</class>
 <widget class="QDialog" name="EditCardDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit card</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="vbox_layout">
     <item>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Card type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="card_types_widget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CompletionComboBox" name="tags">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="button_row">
       <item>
        <widget class="QPushButton" name="OK_button">
         <property name="text">
          <string>&amp;OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="preview_button">
         <property name="text">
          <string>&amp;Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="text">
          <string>&amp;Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompletionComboBox</class>
   <extends>QComboBox</extends>
   <header>mnemosyne/pyqt_ui/completion_combo_box</header>
  </customwidget>
 </customwidgets> 
 <resources>
  <include location="mnemosyne.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>preview_button</sender>
   <signal>clicked()</signal>
   <receiver>EditCardDlg</receiver>
   <slot>preview</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK_button</sender>
   <signal>clicked()</signal>
   <receiver>EditCardDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>EditCardDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
